test/unit/eval_algorithms/test_classification_accuracy_semantic_robustness.py

```diff
@@ -228,270 +225,88 @@ def test_classification_accuracy_semantic_robustness_evaluate_sample(self, test_
         )
         assert model.predict.call_count == 3
 
-    @pytest.mark.parametrize(
-        "test_case",
-        [
-            TestCaseClassificationAccuracySemanticRobustnessEvaluateSample(
-                model_input="Ok brownie.",
-                original_model_output="3",
-                perturbed_model_output_1="Some model output.",
-                perturbed_model_output_2="Some model output.",
-                target_output="3",
-                expected_response=[
-                    EvalScore(name=CLASSIFICATION_ACCURACY_SCORE, value=1.0),
-                    EvalScore(name=DELTA_CLASSIFICATION_ACCURACY_SCORE, value=1.0),
-                ],
-                config=ClassificationAccuracySemanticRobustnessConfig(
-                    valid_labels=["1", "2", "3", "4", "5"],
-                    num_perturbations=2,
-                ),
-            )
-        ],
-    )
-    def test_classification_accuracy_semantic_robustness_evaluate_sample_with_model_output(self, test_case):
```

Comment:
Getting rid of this test case as we no longer have `model_output` as a parameter in `evaluate_sample`. I've done the same thing for QA accuracy semantic robustness. The rationale is that users need to provide the `model` in order to get perturbed responses anyways, so it's just added work for them to get the model output first, and then pass it to the `evaluate_sample` method.